package.json

```diff
@@ -100,6 +100,7 @@
     "@appium/types": "file:packages/types",
     "@appium/universal-xml-plugin": "file:packages/universal-xml-plugin",
     "appium": "file:packages/appium",
+    "chalk": "4.1.2",
```

Comment:
`@appium/support` uses `@colors/colors` instead of `chalk`, so it'd probably be best to stick to that one unless there's a compelling reason to use `chalk` instead?

Reply:
There is no `.hex` in `@colors/colors` I am not against changing to use of the colors available in `@colors/colors` but I don't believe there are 24 evenly spaced colors of the rainbow to create a complete color wheel.

The colors are generated in hex from running `rainbow-colors-array` in a node script and copying the output, it seems like `@colors/colors` will only allow predefined colors.